WordPress/src/main/java/org/wordpress/android/ui/compose/components/VerticalScrollBox.kt

```diff
@@ -0,0 +1,25 @@
+package org.wordpress.android.ui.compose.components
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.BoxScope
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+
+/**
+ * Box UI component with vertical scroll.
+ */
+@Composable
+fun VerticalScrollBox(
+    modifier: Modifier = Modifier,
+    alignment: Alignment = Alignment.TopStart,
+    content: @Composable BoxScope.() -> Unit
+) {
+    Box(
+        modifier = modifier.then(Modifier.verticalScroll(rememberScrollState())),
```

Comment:
I think the `rememberScrollState` has to be outside of the box as a variable. I'm not sure why though. Recomposition?

Reply:
I agree with this point, I think the `ScrollState` should be provided as an argument, to allow clients to control the scrolling if they want, for example, to scroll programmatically or to listen for scroll changes.

Reply:
Thank you both for this suggestion. Not sure I've properly understood both comments, since @hichamboushaba is AFK today I went forward with an approach that should allow clients to control the scrolling if needed by making it an optional parameter of `VerticalScrollBox`.

Let me know if the changes in [48868fe](https://github.com/wordpress-mobile/WordPress-Android/pull/16934/commits/48868fe14e7acbfd870ab4e5d69efb2e20c257ba) satisfy both suggestions for improvement 🙇 .

Reply:
Looks good to me @ovitrif, the only point is that Lint would complain about "Modifier parameter should be the first optional parameter" with this, so I'd suggest moving the `scrollState` below it.

Reply:
I think the last point was solved in a subsequent commit, so all good.